Composer/package.json

```diff
@@ -72,14 +72,16 @@
     "typecheck:client": "yarn workspace @bfc/client typecheck",
     "tableflip": "rimraf node_modules/ **/node_modules && yarn && yarn build",
     "l10n:extract": "cross-env NODE_ENV=production format-message extract -g underscored_crc32 -o packages/server/src/locales/en-US.json l10ntemp/**/*.js",
+    "l10n:makeTSV": "node scripts/l10n-makeTSV.js packages/server/src/locales/en-US.json",
     "l10n:extractJson": "node scripts/l10n-extractJson.js",
     "l10n:transform": "node scripts/l10n-transform.js",
     "l10n:babel": "babel ./packages --extensions \".ts,.tsx,.jsx,.js\" --out-dir l10ntemp --presets=@babel/react,@babel/typescript --plugins=@babel/plugin-proposal-class-properties --ignore \"packages/electron-server\",\"packages/**/__tests__\",\"packages/**/node_modules\",\"packages/**/build/**/*.js\"",
-    "l10n": "yarn l10n:babel && yarn l10n:extract && yarn l10n:transform packages/server/src/locales/en-US.json && yarn l10n:extractJson packages/server/schemas"
+    "l10n": "yarn l10n:babel && yarn l10n:extract && yarn l10n:makeTSV && yarn l10n:transform packages/server/src/locales/en-US.json && yarn l10n:extractJson packages/server/schemas"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged",
+      "pre-push": "yarn l10n"
```

Comment:
Can we instead make this a github action that runs on a cron? This is a little disruptive to the dev experience.

Reply:
This is intended to stop another disruptive behavior of someone getting a PR merged that hasn't had the localization script run followed by the next person to pick up the build wondering why `en-US.json` changes when they haven't done anything to strings. I'll look into making it into an action, though.

Reply:
My suggestion is to stop running the script as a part of normal dev so that it is not disruptive in those ways. The action could run every night and if the en-US.json file changes, it can commit the result directly into main.